### PR TITLE
Fix bug where preinstalled site extension is empty

### PR DESCRIPTION
**Pull request by WilliamXieMSFT**
Without this fix, the preinstalled site extension .zip file can't resolve filepaths.

### CODE REVIEW

File: src/InstrumentationEngine.Preinstall/InstrumentationEngine.Preinstall.csproj [hunk 1]
```diff
@@ -3,6 +3,7 @@
       -->
 <Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), EnlistmentRoot.marker))\build\Common.props" />
+  <Import Project="$(EnlistmentRoot)\build\Packaging.props" />
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
   <PropertyGroup>
```
Comment (wiktork) [inline src/InstrumentationEngine.Preinstall/InstrumentationEngine.Preinstall.csproj]: did you need to version bump?

Comment (WilliamXieMSFT) [reply src/InstrumentationEngine.Preinstall/InstrumentationEngine.Preinstall.csproj]: Luckily not for this change

Comment (WilliamXieMSFT) [reply src/InstrumentationEngine.Preinstall/InstrumentationEngine.Preinstall.csproj]: It appears this only fixes local build but not from ADO. Currently investigating...

**Pull request merged**